README.md

```diff
@@ -29,7 +29,7 @@ Detailed installation instructions can be found [here](https://open-forcefield-t
 
 ### Installing from source
 
-If you need to install via source, see the build and run package requirements listed in the [development conda recipe](https://github.com/openforcefield/openff-toolkit/blob/master/devtools/conda-recipe/meta.yaml).
+If you need to install via source, see the build and run package requirements listed in the [development conda recipe](https://github.com/openforcefield/openff-toolkit/blob/0.8.3/devtools/conda-recipe/meta.yaml).
```

Comment:
This is the best solution I could come up with - this file is no longer tracked in this repo (a good change, in my opinion) and I did not want to add a link to a feedstock repo that does not exist yet. So, I think the best thing to do is pin it to a file from an old tag and update it once it has a new home (sometime between 0.9.0 and 0.9.1). I'll add this note to the issue tracking the migration.